pkg/volume/rbd/rbd_util.go

```diff
@@ -35,17 +35,21 @@ import (
 	"github.com/golang/glog"
 	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/util/wait"
 	fileutil "k8s.io/kubernetes/pkg/util/file"
 	"k8s.io/kubernetes/pkg/util/node"
 	"k8s.io/kubernetes/pkg/volume"
 	volutil "k8s.io/kubernetes/pkg/volume/util"
 )
 
 const (
-	imageWatcherStr = "watcher="
-	imageSizeStr    = "size "
-	sizeDivStr      = " MB in"
-	kubeLockMagic   = "kubelet_lock_magic_"
+	imageWatcherStr          = "watcher="
+	imageSizeStr             = "size "
+	sizeDivStr               = " MB in"
+	kubeLockMagic            = "kubelet_lock_magic_"
+	rbdImageWatcherInitDelay = 1 * time.Second
```

Comment:
nit: add a note here about the total duration

Reply:
done